db/re/item_db.conf

```diff
@@ -6419,7 +6419,7 @@ item_db: (
 		bonus2 bIgnoreDefRate,RC_DemiPlayer,25;
 		bonus bUnbreakableWeapon,0;
 		if(getrefine()>5) {
-			bonus2 bAddRace,RC_DemiPlayer,pow(min(getrefine(), 14)-3,2);
+			bonus2 bAddRace,RC_DemiPlayer,((min(getrefine(), 14)-3) ** (2));
```

Comment:
I was too lazy to go through the whole database so I used `sed`, hence the extraneous parentheses